src/core/dbus-execute.c

```diff
@@ -781,7 +818,7 @@ const sd_bus_vtable bus_exec_vtable[] = {
         SD_BUS_PROPERTY("RuntimeDirectory", "as", NULL, offsetof(ExecContext, runtime_directory), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("MemoryDenyWriteExecute", "b", bus_property_get_bool, offsetof(ExecContext, memory_deny_write_execute), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("RestrictRealtime", "b", bus_property_get_bool, offsetof(ExecContext, restrict_realtime), SD_BUS_VTABLE_PROPERTY_CONST),
-        SD_BUS_PROPERTY("RestrictNamespaces", "t", bus_property_get_ulong, offsetof(ExecContext, restrict_namespaces), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("RestrictNamespaces", "s", bus_property_get_restrict_namespaces, offsetof(ExecContext, restrict_namespaces), SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
hmpf, properties on the bus are so far supposed to be normalized, low-level values and explicitly not user friendly, possibly redundant fields. So I am very much against this.

If anything this kind of prettification should take place on the client side, where we do this already for a number of props (though quite incomplete).

Properties exposed on the bus should primarily be considered an API for other apps to consume, and for that a flags field is definitely the nicer and more natural concept. If "systemctl show" then wants to expose this more friendly for shell scripts, then it should convert this as necessary...